Sites/GD2/Project 1/Styles.css

```diff
@@ -0,0 +1,50 @@
+body {
+    margin: 100px;
+}
+
+h1 {
+    font-family: 'Noto Sans TC', sans-serif;
+    color: #260648;
+    font-weight: bold;
+    font-size: 45px;
+}
+
+h2 {
+    font-family: 'Noto Sans TC', sans-serif;
+    color: #6B4B8C;
+    font-weight: normal;
+    font-size: 30px;
+    text-align: left;
+}
+
+div {
+    margin: 40px;
+}
+
+blockquote {
+    font-family: 'PT Sans', sans-serif;
+    font-weight: italic;
+    font-size: 30px;
+    color: #6B4B8C;
+}
+
+a {
+    color: #EE3595;
+}
+
+p {
+    font-family: 'PT Serif', serif;
+    font-weight: normal;
```

Comment:
The default `font-weight` for a `p` element is normal, so you can safely delete this line.